app/routes/events/components/EventEditor/index.js

```diff
@@ -382,6 +382,19 @@ function EventEditor({
                 />
               </Tooltip>
             )}
+            {['NORMAL', 'INFINITE'].includes(event.eventStatusType) && (
+              <Tooltip content="Navn og epost kan deles med folk utenfor Abakus til smittesporing">
```

Comment:
Should probably add phone number to the tooltip as well

Reply:
As well as an instruction to specify in the event description who the list will be shared with.